sunbeam-python/sunbeam/utils.py

```diff
@@ -374,6 +374,17 @@ def decorator(func: click.decorators.FC) -> click.decorators.FC:
     return decorator
 
 
+def click_option_show_hints(func: click.decorators.FC) -> click.decorators.FC:
+    """Common decorator to show hints for questions."""
+    return click.option(
+        "--hints/--no-hints",
+        "show_hints",
+        is_flag=True,
+        default=True,
```

Comment:
Isn't it better for default False and user can set --hints if more info is required?

Reply:
We had this discussion, and I believe the interactive installation is tailored towards first time users. Veterans will most likely use the manifest and not use the interactive CLI